.github/CODEOWNERS

```diff
@@ -1,27 +1,64 @@
 # Code owners file.
 # This file controls who is tagged for review for any given pull request.
+#
+# For syntax help see:
+# https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
+
 
 # The python-samples-owners team is the default owner for anything not
 # explicitly taken by someone else.
-*                         @GoogleCloudPlatform/python-samples-owners
-
-# Alix Hamilton is the primary maintainer of the BigQuery samples.
-bigquery/*                @alixhami
-
-# Tim Swast is the primary maintainer of the BigQuery Data Transfer samples.
-bigquery/transfer/*       @tswast
-
-# Tim Swast is the primary maintainer of the Composer samples.
-composer/*                @tswast
-
-# Gus Class is the primary maintainer of the IoT samples.
-iot/*                     @gguuss
-
-# Alix Hamilton is the primary maintainer of the Jupyter notebook samples.
-notebooks/*               @alixhami
-
-# Torry Yang is the primary maintainer of the Tables samples.
-/tables/                  @sirtorry
+*                       @GoogleCloudPlatform/python-samples-owners
```

Comment:
One issue we'll have to resolve at some point is how to handle reviewers from two different sets we discussed at the last meeting:

> ```
> # Order is important; the last matching pattern takes the most
> # precedence. When someone opens a pull request that only
> # modifies JS files, only @js-owner and not the global
> # owner(s) will be requested for a review.
> *.js    @js-owner
> ```

So this means that currently any changes to just the samples won't trigger/require a review from the "python samples readability" set. 

One work around to this might be to d something like this for each entry:
```
/cloud-sql/                     @kvg @GoogleCloudPlatform/python-samples-owners
```

Reply:
Done